src/handlers/dns_xact/README.md

```diff
@@ -0,0 +1,7 @@
+# DNS Stream Handler
```

Comment:
Should this be the "DNS Transaction (xact) Stream Handler"?

Reply:
Yes, it should

Reply:
done